scripts/bootstrap.sh

```diff
@@ -88,7 +88,7 @@ echo "
 " >>$ACCOUNTS_FILE
 echo "" >>$ACCOUNTS_FILE
 
-cord='./target/release/cord'
+cord="sudo docker run -v $OUTPUT_DIR:$OUTPUT_DIR dhiway/cord:develop"
```

Comment:
can you have the check for building OUTPUT_DIR as the absolute path, instead of relative path?